choreographer/src/main/java/eu/arrowhead/core/choreographer/service/ChoreographerDriver.java

```diff
@@ -121,4 +137,12 @@ private UriComponents getUnregisterSystemUri(final String systemName, final Stri
 
         throw new ArrowheadException("Choreographer can't find Service Registry Unregister System URI.");
     }
+    
+    //-------------------------------------------------------------------------------------------------
+    private UriComponents getQueryExecutorServiceInfoUri(final String address, final int port, final String baseUri) {
+    	logger.debug("getExecutorServiceInfoUri started...");
+    	
+    	final String scheme = sslProperties.isSslEnabled() ? CommonConstants.HTTPS : CommonConstants.HTTP;
```

Comment:
are we sure about this? If the chorepgrapher runs in secure mode, then every executor does the same? Let's talk about this.

Reply:
Since executors are application systems, they have to be able to prove they are part of the cloud. At least I see this way

Reply:
Yeah, it makes sense.